core/jobs/src/main/java/com/cognifide/aet/job/common/modifiers/waitfor/WaitForHelper.java

```diff
@@ -0,0 +1,27 @@
+package com.cognifide.aet.job.common.modifiers.waitfor;
+
+import com.cognifide.aet.communication.api.metadata.CollectorStepResult;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.FluentWait;
+
+import java.util.NoSuchElementException;
+import java.util.concurrent.TimeUnit;
+
+public class WaitForHelper {
```

Comment:
Since this class is just a helper it could be `final` with `private` constructor.